docs/groovy-script/mods/mwc/crafting_station.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Crafting Station/Ammo Press"
```

Comment:
why is this called two things? it is only modifying one set of recipes, it should be named accordingly. if those recipes can happen in multiple places, thats a job for the description, not the title

Reply:
It's 2 different block using same mechanism. BULLET type of recipe can only be accessed by Ammo Press, which is separate block.

Reply:
alright, so name it something that isnt either of those options and indicate that "BULLET is in Ammo Press and everything else is in Crafting Station"

Reply:
BULLET, MAGAZINE, and GRENADE are all in the ammo press 

GUN, ATTACHMENT_NORMAL, and ATTACHMENT_MODIFICATION are in the workbench